README.md

```diff
@@ -8,6 +8,10 @@ Effortless migration
 
 This repo will contain code and documentation to assist in migrations and upgrades of OpenSearch clusters.
 
+## Setup for commits
+
+Developers must run the "setup.sh" script in order to add the pre-commit hook.
```

Comment:
I would probably change the name of this file from setup.sh to install_githooks.sh  or a name that describes exactly what this is doing. setup.sh is often used for building and that is not the case here.

Reply:
@okhasawn can you update this documentation for the new script name?